packages/node/src/storage/NodeFileSystem.ts

```diff
@@ -0,0 +1,62 @@
+import { BacktraceAttachment, FileSystem } from '@backtrace-labs/sdk-core';
+import fs from 'fs';
+import { BacktraceFileAttachment } from '../attachment';
+
+export class NodeFileSystem implements FileSystem {
+    public readDir(dir: string): Promise<string[]> {
+        return fs.promises.readdir(dir);
+    }
+
+    public readDirSync(dir: string): string[] {
+        return fs.readdirSync(dir);
```

Comment:
previously you were using the following options to read the dir:

```
  const databaseFiles = fs.readdirSync(databasePath, {
            encoding: 'utf8',
            withFileTypes: true,
        });
```

Are you sure it works correctly without them?

Reply:
yes

Reply:
How so - we had fun use cases with our customers in India/asia having issues with default encoding. I think using utf8 is necessary. is this a default option?

Reply:
`utf-8` is the default encoding in `readdir`/`readdirSync`